checker/designatorresolution/designator_resolution.go

```diff
@@ -23,7 +22,7 @@ func RunDesignatorResolution(symTable *symbol.SymbolTable) []error {
 
 func (dr *designatorResolution) resolveDesignators() {
 	contractSymbol := dr.symTable.GlobalScope.Contract
-	v := visitor.NewDesignatorResolutionVisitor(dr.symTable, contractSymbol)
+	v := NewDesignatorResolutionVisitor(dr.symTable, contractSymbol)
```

Comment:
`NewDesignatorResolutionVisitor` function can be private now, can't be?

Fix it for all New visitor methods.